index.js

```diff
@@ -1,27 +1,29 @@
 const express = require("express");
-const bodyParser = require("body-parser");
 require("dotenv").config();
 const scrapeController = require("./controllers/webscrapeController");
-
-const app = express();
+const { getDateRange } = require("./models/dates");
 const mongoose = require("mongoose");
-const todoRoutes = require("./routes/todo.js");
 
-main()
-  .then((res) => console.log(res))
-  .catch((err) => console.log(err));
+mongoose.connect(process.env.DB);
+console.log("Connected to the database");
+
+const app = express();
 
-async function main() {
-  await mongoose.connect(process.env.DB);
-}
+app.use("/api/todo", require("./routes/todo.js"));
 
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
+app.get("/espn/getGamesByDates", (req, res) => {
+  /**
+   * User can specify startDate and endDate by appending them as query parameters:
+   * http://localhost:8080/espn/getGamesByDates?startDate=20230901&endDate=20230905
+   */
+  const userStartDate = req.query.startDate;
```

Comment:
this api is only to get future games, so start date is always 1 day from today. Users should not be setting start date.
endDate should not be set for any time before 1 day from today -- if it is, throw an error